depthai_nodes/node/utils/detection_config_generator.py

```diff
@@ -49,11 +45,9 @@ def generate_script_content(
             rect.size.height = rect.size.height + {padding} * 2
             rect.angle = 0
 
-            cfg.addCropRotatedRect(rect=rect, normalizedCoords=True)
-            cfg.setOutputSize({resize_width}, {resize_height})
+            cfg.addCropRotatedRect(rect, True)
```

Comment:
Nitpick: Let's add `normalizedCoords` back as keyword for better readability
```suggestion
            cfg.addCropRotatedRect(rect, normalizedCoords=True)
```